### DISCUSSION TITLE
DPMS toggle

### DISCUSSION THREAD

**Discussion by ThomChroma** [hyprwm/Hyprland#2398] [General]
On sway I used to have a keybind so that I could put to sleep my second monitor when I didn't use it. This way the GPU can switch to power saving mode (it's disabled when 2 monitors are active) or I'm not bothered by the light when I'm watching a video in fullscreen or playing a game.

With Hyprland I can have one keybind to switch it on `exec, hyprctl dispatch dpms on DVI-D-1` and another one to switch it off `exec, hyprctl dispatch dpms off DVI-D-1`. A `dpms toggle` or `toggledpms` could be nice to have only one keybind.

But the doc says about dpms `Do not use with a keybind directly.`, is it exactly what I'm doing and shouldn't? 😅 (if yes why?)

**Comment by txtyash** [comment]
Even I was wondering why my laptop was unresponsive after `hyprctl dispatch dpms off`. I should also add a `bindl` to turn it on.

**Comment by ThomChroma** [comment]
Fixed with 0.28.0, `dispatch dpms toggle` has been added! 🥳👌
